taskweaver/memory/attachment.py

```diff
@@ -15,17 +15,12 @@ class AttachmentType(Enum):
 
     # CodeInterpreter - generate code
     thought = "thought"
-    python = "python"
-
-    # CodeInterpreter - sample code
-    sample = "sample"
+    reply_type = "reply_type"
+    reply_content = "reply_content"
 
```

Comment:
a bit curious do we still support generating samples with code interpreter or expecting the example generated to be in reply type?

Previously there seems to be three types of generated content:

* text: regular plain text, and will show as text in message of a post 
* example: code block, will show as code attachment, but not actually excute it
* python: code block, will show as code attachment during generation, and execute it, then removed from UI with the execution result shown insted.

Currently seems only text and python remaining.

Reply:
I decided to remove it for two reasons: 1. I rarely see the need to generate sample in practice. 2. It could be handled with text already.